### DISCUSSION TITLE
new uuid column

### DISCUSSION THREAD

**Discussion by aksonnic** [spatie/laravel-medialibrary#2134] [Q&A] [answered]
What's the purpose of the new required uuid column? it does not seem to be used anywhere in the package.

**Comment by freekmurze** [comment, marked as answer]
It will be used in the upcoming pro version of the package. https://medialibrary.pro

**Reply by aksonnic** [reply]
Ugh.

**Reply by freekmurze** [reply]
Why the negative sentiment? You can just remove it if you don't plan to use it.

**Reply by rizaldywirawan** [reply]
it asks for the uuid column value, is there a way to ignore it?
